examples/step-18/doc/intro.dox

```diff
@@ -193,14 +193,29 @@ such that
   (\mathbf{f}, \varphi)_{\Omega(t_{n-1})}
   -(\sigma^{n-1},\varepsilon(\varphi))_{\Omega(t_{n-1})}
   \\
-  &\qquad +(\mathbf{b}(\mathbf{x},t_n)-\mathbf{b}(\mathbf{x},t_{n-1}), \varphi)_{\Gamma_N}
+  &\qquad 
+  +(\mathbf{b}(\mathbf{x},t_n)-\mathbf{b}(\mathbf{x},t_{n-1}), \varphi)_{\Gamma_N}
+  +(\sigma^{n-1} \mathbf{n}, \varphi)_{\Gamma_N}
+  \\
+  &\qquad\qquad
+  \forall \varphi \in \{\mathbf{v}\in H^1(\Omega(t_{n-1}))^d: \mathbf{v}|_{\Gamma_D}=0\}.
+@f}
+Using that $\sigma^{n-1} \mathbf{n} = \mathbf{b}(\mathbf x, t_{n-1})$,
```

Comment:
Could you clarify why this holds?

Reply:
That's just the Neumann boundary condition stated further up ahead: 
https://github.com/dealii/dealii/blob/master/examples/step-18/doc/intro.dox#L56

Reply:
But the Neumann boundary condition is `\mathbf{n} \varepsilon(\mathbf{u}(\mathbf{x},t))` and doesn't involve `\sigma`. The only identification we make between `\mathbf{u}` and `\sigma` is `\dot\sigma = C \varepsilon (\dot{\mathbf{u}})`.